test/e2e/common/volumes.go

```diff
@@ -137,11 +127,9 @@ var _ = Describe("[sig-storage] GCP Volumes", func() {
 			config, _, _ := framework.NewGlusterfsServer(c, namespace.Name)
 			name := config.Prefix + "-server"
 			defer func() {
-				if clean {
-					framework.VolumeTestCleanup(f, config)
-					err := c.CoreV1().Endpoints(namespace.Name).Delete(name, nil)
-					Expect(err).NotTo(HaveOccurred(), "defer: Gluster delete endpoints failed")
-				}
+				framework.VolumeTestCleanup(f, config)
+				err := c.Core().Endpoints(namespace.Name).Delete(name, nil)
```

Comment:
CoreV1

Reply:
thanks,I amend the error and wait for your review again ,for your lgtm!thanks very much!